v22.1/deploy-app-vercel.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Deploy a Web App Built on CockroachDB with Vercel
+summary: Learn how to use Vercel and CockroachDB Serverless.
+toc: true
+twitter: false
+referral_id: docs_vercel
+docs_area: get_started
+---
+
+This tutorial shows you how to use [Vercel](https://vercel.com/) to deploy a web application built with [Next.js](https://nextjs.org/), [Express](https://expressjs.com/), [Prisma](https://www.prisma.io/), and {{ site.data.products.serverless }}.
+
+{{site.data.alerts.callout_success}}
+For more detailed instructions on building a fullstack app with Next.js and Prisma, see Vercel's official [How to Build a Fullstack App with Next.js, Prisma, and PostgreSQL](https://vercel.com/guides/nextjs-prisma-postgres) guide.
+{{site.data.alerts.end}}
+
+## Prerequisites
+
+Before starting the tutorial, do the following:
+
+1. Create a [{{ site.data.products.db }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account.
+
+1. Create a [Vercel](https://vercel.com/signup) account. You can do this with your GitHub login credentials.
```

Comment:
I'd remove the "GitHub" portion since I don't think we have a reason to prefer that users use any particular login method.